save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/internal/TestSuitesController.kt

```diff
@@ -0,0 +1,75 @@
+package com.saveourtool.save.backend.controllers.internal
+
+import com.saveourtool.save.backend.configs.ApiSwaggerSupport
+import com.saveourtool.save.backend.service.TestSuitesService
+import com.saveourtool.save.entities.TestSuite
+import com.saveourtool.save.testsuite.TestSuiteDto
+import com.saveourtool.save.utils.blockingToMono
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.Parameters
+import io.swagger.v3.oas.annotations.enums.ParameterIn
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import io.swagger.v3.oas.annotations.tags.Tags
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+
+/**
+ * Controller for test suites
+ */
+@ApiSwaggerSupport
+@Tags(
+    Tag(name = "test-suites"),
+    Tag(name = "internal"),
+)
+@RestController
+@RequestMapping("/internal/test-suites")
```

Comment:
just a note: we do not generate swagger info for `/internal` endpoints

Reply:
Yes, but as far as I remember we decided to write comments for controller methods in this style, but not in kdoc

Reply:
I don't remember that we agreed on it, I vote against it 😄

Reply:
Oh ok